rand_core/src/error.rs

```diff
@@ -9,13 +9,13 @@
 //! Error types
 
 use core::fmt;
-
-#[cfg(feature="std")]
-use std::error::Error as stdError;
-#[cfg(feature="std")]
-use std::io;
+#[cfg(not(feature="std"))]
+use core::num::NonZeroU32;
 
 
+// A randomly-chosen 24-bit prefix for our codes.
+#[cfg(not(feature="std"))]
+pub(crate) const CODE_PREFIX: u32 = 0x517e8100;
```

Comment:
Why do we need this?

Reply:
The current `no_std` code uses a code, but one isn't always provided. This is in line with what happens in `getrandom`, but may not be optimal.